develop/ThirdParty/Grafana/Grafana_plugin.md

```diff
@@ -26,7 +26,9 @@ The Grafana Data Source plugin allows you to visualize data from DataMiner in [G
 
    - **Windows**: Go to `C:\Program Files\GrafanaLabs\grafana\conf`. If there is no `custom.ini` file yet, copy `sample.ini` and rename it to `custom.ini`. This `custom.ini` file is your configuration file.
 
-   - **Linux**: Go to `$WORKING_DIR/conf`. If you find the `defaults.ini` file, you have located your Grafana configuration file. If there is no `defaults.ini` file, go to `/usr/local/etc/grafana` and find `grafana.ini`. In this case, `grafana.ini` is your configuration file.
+   - **Linux**: Go to `/etc/grafana/grafana.ini` if you installed Grafana using the `deb` or `rpm` packages. The `grafana.ini` file is your configuration file.
+
+   - **macOS**: Go to `/usr/local/etc/grafana/grafana.ini`. If you installed your Grafana instance using Homebrew, `grafana.ini` is your configuration file. Otherwise, add a configuration file named `custom.ini` to the *conf* folder.
```

Comment:
- Do users just add an empty file named "custom.ini"? Or should they copy an existing file and rename it, similar to the step for Windows?
- Could you add the path to the conf folder? I would like to rephrase this step so that it is similar to the Windows instructions.

Reply:
To be honest I'm not really sure. Maybe it's safer to not document this and instead put a link to https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/

Reply:
I have indeed put a link to the Grafana documentation on how to locate the configuration file. This is also more update-proof.

Reply:
Thanks! Looks good to me to merge this.